File: FirstHomework/app/src/main/java/com/example/firsthomework/MainActivity.java
package com.example.firsthomework;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    Button button;
    TextView textview;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        button = findViewById(R.id.button);
        button.setOnClickListener(x -> onClick());
        textview = findViewById(R.id.textView1);
        textview.setOnClickListener(x -> onClick2());
    }

    public void onClick() {
        Intent intent = new Intent(this, ThirdScreen.class);
        startActivity(intent);
    }

    public void onClick2() {
        Intent intent = new Intent(this, SecondScreen.class);
        startActivity(intent);
    }
}

File: It/app/src/main/java/com/example/it/MainActivity.kt
package com.example.it

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val richie = OneOfLoserClub("Richie Tozier", "Paul Banyan", null, 13)
        val bev = OneOfLoserClub("Beverly Marsh", "Blood", null, 12)
        val pennywise = Pennywise("physical form", 100,"Clown")
        val henry = BadGuy("Henry Bowers", "knife", null)
        val fight = Fight()
        fight.fightWithBadGuy(richie, henry)
        fight.fightWithIt(bev, pennywise)

    }
}

inline fun printMessage(message: String){
    Log.e("Done with it", message)
}